x-pack/plugins/index_lifecycle_management/public/app.js

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { HashRouter, Switch, Route } from 'react-router-dom';
+import { EditPolicy } from './sections/edit_policy';
+import { PolicyTable } from './sections/policy_table';
+import { BASE_PATH } from '../common/constants';
+
+export const App = () => (
+  <HashRouter>
+    <Switch>
+      <Route exact path={`${BASE_PATH}policies`} component={PolicyTable}/>
```

Comment:
clicking `Index Lifecycle Management` from the top nav breadcrumb goes to `#/management/elasticsearch/index_lifecycle_management`, which is blank. add redirect to fix:

```suggestion
      <Route exact path={`${BASE_PATH}policies`} component={PolicyTable}/>
      <Redirect exact from={`${BASE_PATH}`} to={`${BASE_PATH}policies`} />
```